ui/components/ui/icon-button/icon-button.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { IconColor } from '../../../helpers/constants/design-system';
+import { ICON_NAMES, Icon } from '../../component-library';
+import IconButton from '.';
+
+export default {
+  title: 'Components/UI/IconButton',
+  component: IconButton,
+  argTypes: {
+    onClick: { action: 'clicked' },
+    Icon: {
+      control: 'node',
+    },
+    disabled: {
+      control: 'boolean',
+    },
+    label: {
+      control: 'text',
+    },
+    tooltipRender: {
+      control: 'function',
+    },
+    className: {
+      control: 'text',
+    },
+  },
+  args: {
+    Icon: <Icon name={ICON_NAMES.CARD} color={IconColor.primaryInverse} />,
+    label: 'Buy',
+  },
+};
+
+export const DefaultStory = (args) => <IconButton {...args} />;
+
+DefaultStory.storyName = 'Default';
```

Comment:
Adding IconButton story for easier future visual updates

<img width="968" alt="Screenshot 2023-03-05 at 5 43 10 PM" src="https://user-images.githubusercontent.com/8112138/223001044-fc0243da-1fa0-4391-a722-23eb0209f286.png">